librz/bp/bp.c

```diff
@@ -415,3 +416,56 @@ RZ_API bool rz_bp_is_valid(RzBreakpoint *bp, RzBreakpointItem *b) {
 
 	return bp->corebind.isMapped(bp->corebind.core, b->addr, b->perm);
 }
+
+/**
+ * @brief Set values of members of a RzBreakpointItem
+ * 
+ * @param item breakpoint item to set values for
+ * @param cond value of cond to be set
+ * @param data value of data to be set
+ * @param expr value of expr to be set
+ * @param name value of name to be set
+ * @return bool true if succesful; false otherwise; in case false is returned, none of the values will have been modified
+ */
+RZ_API bool rz_bp_item_set(RZ_NONNULL RzBreakpointItem *item, RZ_NULLABLE const char *cond, RZ_NULLABLE const char *data, RZ_NULLABLE const char *expr, RZ_NULLABLE const char *name) {
```

Comment:
I'm not sure I like this kind of API. Rather than this i prefer one API for each field...

Reply:
Yes yes. I will split them. Just couldn't do it yesterday, because it was late.